docs/DesignDocs/InitTimeConfigurationInterface.md

```diff
@@ -114,7 +152,8 @@ enclave developer.
 
 On SGX CPUs supporting the KSS feature, `CONFIGID` and `CONFIGSVN` are available
 in the SGX `REPORT`. The OE SDK libs will provide an API to retrieve config_id
-and config_svn within the enclave.
+and config_svn within the enclave. The [code commit](https://github.com/openenclave/openenclave/commit/7bb04b51fdf67a55d927921891473d7521f979ec#diff-8ddfddac7b1a84829881b6bc97d2b812dfbb7693a143ae6cb033285d857ce636) demonstrates how developers can use APIs to retrieve config_id and config_svn within the enclave
```

Comment:
Is it acceptable to link to a commit? Can be a link to a file in repo instead?

Reply:
As file content could be changing any time, commit id provides static code.

Reply:
This is an important example too. We can add a code snippet? I'd prefer an example based on `oe_get_report_v2` and `oe_parse_report`.

Reply:
In my opinion: we should never be linking to a commit.  I think it should point to something (e.g., a markdown file, or a sample, or whatever) in the latest master.   If it changes in a way that makes this statement false, I would say the PR that makes the change ought to update this text.